src/services/mongoose/schema.ts

```diff
@@ -285,6 +285,8 @@ const AccountSchema = new Schema<AccountRecord>(
       type: String,
       index: true,
     },
+
+    displayCurrency: String, // FIXME: should be an enum
```

Comment:
Needs to be added to AccountRecord type as well?

Reply:
already there